### ISSUE TITLE
Export Client class declaration

### ISSUE THREAD

**Issue by fi4sk0** [draftable/compare-api-node-client#12]
Hi there, 

might I trouble you to also export the Client class in https://github.com/draftable/compare-api-node-client/blob/35a043b155e46d2d66da2e41c997181ecd51c871/src/index.ts#L8 ?

My use case is that I'd like to have the instantiated client as a typed property instead of using "any".

```typescript
export class DraftableComparison extends Service {

  client: any // <-- I'd like to put "Client" here, but it's not exported

  constructor(options: Partial<ServiceOptions>) {
    super(options);

    const config = this.app.config.get('draftable')

    this.client = client(config.accountId, config.authToken)
  }
// ... more code
```

Thanks for your time!

**Comment by Dpio** [comment]
Hey, 

Thanks for using draftable, I've recorded an issue and will schedule the improvement.

Best regards,
Piotr

**Comment by Dpio** [comment]
Hey @fi4sk0, 
a new version was published, please update to v1.4.4.
